added try catch to callApi + alert for catch statement

IhorStorozhok opened the pull request:
https://web.tracklify.com/project/2b7ZVgE5/AdminForth/354/cQ3CFUag/if-back-crashes-with-500-butto

I am not sure , coz u asked about handling only 500 status. By I think it would be better to show alert if any error will be catched in callApi function.

ivictbor commented on adminforth/spa/src/utils.ts:
```diff
@@ -23,13 +23,19 @@ export async function callApi({path, method, body=undefined}: {
+    } 
+    return await r.json();
+  } catch(e){
+    window.adminforth.alert({variant:'danger', message:'Something went wrong, please try again later',})
```
@IhorStorozhok Can you please wrap `message:'Something went wrong, please try again later'` with 't':

```
message: t('Something went wrong, please try again later')
```

IhorStorozhok replied:
Got it. It makes sense. Will add translation function to this and next PRs.

ivictbor commented on adminforth/spa/src/utils.ts:
```diff
@@ -23,13 +23,19 @@ export async function callApi({path, method, body=undefined}: {
+    return await r.json();
+  } catch(e){
+    window.adminforth.alert({variant:'danger', message:'Something went wrong, please try again later',})
+    console.error('error',e);
```
@IhorStorozhok can we also rename 'error' -> `error in callApi ${path}` ?

IhorStorozhok replied:
Sure. Good point.

IhorStorozhok commented:
I've added translations but was forced to add i18n.instance to window object  to get access for i18n outside the components (utils.ts) . window.adminforth is not initialised in this time , thats why i added i18n directly to window objtct.
